source/gui/settingsDialogs.py

```diff
@@ -3624,6 +3624,84 @@ def onSave(self):
 				_remoteClient.terminate()
 
 
+class LocalCaptionerSettingsPanel(SettingsPanel):
+	"""Settings panel for Local captioner configuration.
+
+	This panel allows users to configure the local model path and
+	initialization settings for the local captioner.
+	"""
+
+	# Translators: This is the label for the local captioner settings panel.
+	title = _("Local Captioner")
```

Comment:
Can we change the phrasing to "AI Image Descriptions"

And "ImageDescriber" rather than "LocalCaptioner"

```suggestion
	title = _("AI Image Descriptions")
```